stripe/src/main/java/com/stripe/android/googlepay/StripeGooglePayContract.kt

```diff
@@ -40,7 +41,8 @@ internal class StripeGooglePayContract :
     @Parcelize
     data class Args(
         var paymentIntent: PaymentIntent,
-        var config: GooglePayConfig
+        var config: GooglePayConfig,
+        @ColorInt val statusBarColor: Int?
```

Comment:
`= null` here? so that the changes in `DefaultFlowControllerTest` and `StripeGooglePayViewModelTest` can be voided

Reply:
The tradeoff is then it's easy to forget to pass it, which I don't want.